src/common/components/gear-menu-button.tsx

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { NamedFC } from 'common/react/named-fc';
+import { IconButton, IContextualMenuItem } from 'office-ui-fabric-react';
+import * as React from 'react';
+import { DropdownClickHandler } from '../dropdown-click-handler';
+import { FeatureFlags } from '../feature-flags';
+import { FeatureFlagStoreData } from '../types/store-data/feature-flag-store-data';
+import * as styles from './gear-menu-button.scss';
+
+export interface GearOptionsButtonComponentProps {
```

Comment:
We can rename this to reflect the new component name

Reply:
Good catch